src/sass/modules/_m-footer.scss

```diff
@@ -19,7 +19,94 @@
   }
 }
 
-  .footer-inner {
+.feedback-widget {
+  padding: 1.5em 0;
+  background-color: $color-gray-lightest;
+  color: $color-gray-dark;
+
+  .row {
```

Comment:
This is something that might trip someone up if we make design changes in there again later and they need to use `row` in a different context.

Reply:
Yeah good call. Also, unless I overlooked something, we have no simple wrapper class for just adding something to the page that lines up with the rest of the elements, which is why this has that ugly padding property. It looks like we got mixed up between USWDS, Foundation, and our own classes.